tests/unit/test_database.py

```diff
@@ -369,7 +255,3 @@ def create_upsert_rules_in_db(cls):
     def create_recursive_functions_in_db(cls):
         config.db_session.execute(cls.get_all_children_function_sql + cls.get_all_parents_function_sql)
         config.db_session.commit()
-
-
-if __name__ == '__main__':
```

Comment:
Why is this deleted? It is required to be able to run the file directly.

Reply:
👍

Reply:
I think it just ended up in a different file when I split the test files